### ISSUE TITLE
Runtime error on command line only - cannot parse nodes

### ISSUE THREAD

**Issue by thejeff77** [jasonruesch/xlifftranslate#1]
Issue running the translator:
```

ath fr.xcloc/Localized\ Contents/
/usr/local/lib/node_modules/xlifftranslate/bin/xlifftranslate.js:105
            translate.translate(text.replace('<x id="INTERPOLATION"/>', '<_______>'), lang).then(function (results) {
                                     ^

TypeError: Cannot read property 'replace' of null

```

Running this in a debugger in Webstorm seems to work fine, but running it from the console yields 'null' for each iteration grabbing the text var. I'm not sure what is happening there.

**Comment by thejeff77** [comment]
This one is super weird, I have no idea what is going on. Looks like cheerio has 177 bugs... When I run this on the command line printing out the value of the 'text' variable, my output is:

```
null
null
null
```

etc.. 

Running this in an IDE works just peachy. 

Are you able to run locally without issue on the current version? I'm using node 8.11.3 in case it matters

Upgrading cheerio to the latest version didn't fix the issue.

**Comment by jasonruesch** [comment]
> Are you able to run locally without issue on the current version? I'm using node 8.11.3 in case it matters

Hi @thejeff77! I am able to run it successfully in the terminal. I'm using node 8.12.0. I am not sure why cheerio would be giving you issues.

Are you still having problems?

**Comment by thejeff77** [comment]
Yes, although different problems. I'm determined to figure this out and add some options. I hope you're open to pull requests, as this is a cool tool and I want to continue using it for some xcode translation use cases. 

I found that some of my issues were with the first line's definition of node. This isn't typical, and usage with nvm and/or cross platform doesn't guarantee location of node - so we'd want to instruct the caller to run with '>node xlifftranslate' to run the node version that is available in the environment rather than a vanilla call to '>xlifftranslate'

Still playing with it!

**Comment by jasonruesch** [comment]
Awesome! Glad someone is interested in this. Please, feel free to create pull requests!
